src/components/CommitteeTools/index.scss

```diff
@@ -1,141 +1,61 @@
-@import "../../styles/constants.scss";
+@import "styles/constants.scss";
 
-$committee-blue: #C0F2F2;
+$committee-blue: #c0f2f2;
```

Comment:
Please don't add color variables here. First search for it here `src/styles/constants.scss`, if you can't find so add it there.

Reply:
ok